contents/tutorials/beta-feedback.md

```diff
@@ -0,0 +1,135 @@
+---
+title: How to collect beta feedback from beta users
+date: 2023-11-07
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+tags: ['surveys', 'feature flags']
+---
+
+The goal of a beta is to get a feature ready for a full release. This means fixing what isn’t working. Getting beta user feedback is one of the best ways to know what to fix, and PostHog makes it easy to collect.
```

Comment:
Also "Getting beta user feedback is one of the best ways to know what to fix, and PostHog makes it easy to collect." is missing surveys I think e.g.

"Using surveys to request feedback from your beta users is one of the best ways to know what to fix, and PostHog makes it easy to collect."

Reply:
rewrote to better frame this way